javascript/javascript1/week3/notes.js

```diff
@@ -0,0 +1,44 @@
+const notes = [];
+
+function saveNote(content1, id1) {
```

Comment:
Think about the naming one more time. I think `content` and `id` would be better than `content1` and `id1` as parameters.

Reply:
Done.Thank you for feedback TITHI .